circle.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
 
 test:
   override:
-    - shellcheck bin/docker-* bin/ensure-kubectl bin/k8s-apply bin/k8s-secrets-from-s3 bin/verify-deployment bin/k8s-lint
+    - shellcheck bin/docker-* bin/ensure-kubectl bin/k8s-apply bin/k8s-secrets-from-s3 bin/verify-deployment bin/k8s-lint bin/cloud-login
```

Comment:
```
~/work/k8s-scripts (gcp-additions)🦄
# shellcheck bin/prepare-awscli

In bin/prepare-awscli line 8:
  eval "$(aws ecr get-login --region ${AWS_DEFAULT_REGION})"
                                     ^-- SC2086: Double quote to prevent globbing and word splitting.


In bin/prepare-awscli line 10:
  eval "$(aws ecr get-login --region ${AWS_DEFAULT_REGION} --registry-ids ${AWS_ECR_ACCOUNT_ID})"
                                     ^-- SC2086: Double quote to prevent globbing and word splitting.
                                                                          ^-- SC2086: Double quote to prevent globbing and word splitting.
```

Do you know how to get around that?